buildconf

```diff
@@ -263,7 +310,17 @@ if test ! -x "$libtoolize"; then
 fi
 if test -z "$libtoolize"; then
   if [[ "$OSTYPE" == "darwin"* ]]; then
-    brew install libtool
+    read -p "Do you wish to install libtool using Homebrew ? y/n `echo $'\n> '`" yn
+     case $yn in 
+        [Yy]* )        
+          brew install libtool
+          break;;
+        [Nn]* ) 
+          echo "buildconf: libtoolize not found."
+          echo "  You need GNU libtoolize $want_lt_version or newer installed."
+          exit 1;;    
+        * ) echo "y for Yes and n for No";
+      esac
```

Comment:
Why not add all the missing pieces as the script goes through and then install all of the missing ones *at once* instead of doing it one by one?

Reply:
Just didn't want to reconstruct it to not make you confused, as first porposal 

So the idea is accepted. I can go ahead and make it at once instead